t/var.t

```diff
@@ -60,3 +60,41 @@ else {
     for (7..8) {print "ok $_  # skipped: not Unicode-aware Perl or Tcl\n";}
 }
 
+# array tie test
+{
+  my @ary = ();
+  tie @ary, "Tcl::AList", $i, "t_list";
+  $i->Eval('set t_list [ list a "b c" "d e f" ]');
+  print "not " unless(@ary == 3);
+  print "ok 9 list has correct length\n";
+  print "not " unless($ary[0] eq 'a' && $ary[1] eq 'b c' && $ary[2] eq 'd e f');
+  print "ok 10 list has correct content\n";
+  my $x = shift(@ary);
+  print "not " unless($x eq 'a');
+  print "ok 11 shift return the first element\n";
+  $x = $i->Eval('concat "$t_list" ""');
```

Comment:
If the purpose of `$i->Eval('concat "$t_list" ""')` is to retrieve the contents of `$t_list` in Tcl, then wouldn't it be clearer to just use `$i->Eval('set $t_list')` or better yet `$i->GetVar('t_list')`?